runtime/src/vm/memory.rs

```diff
@@ -9,6 +8,9 @@ use crate::vm::program::Program;
 // -2^63
 pub const CLOSURE_ARG_MEM_START: i64 = -9223372036854775808;
 pub const CLOSURE_ARG_MEM_SIZE: usize = 4;
+const NORMAL_ADDR: i8 = 0;
+const FX_PTR_ADDR: i8 = 1;
+const HM_PTR_ADDR: i8 = 2;
```

Comment:
nit: can we add a small comment of how each works?

Reply:
Sure.